rollingpin/status.py

```diff
@@ -0,0 +1,65 @@
+import json
+import logging
+import posixpath
+import urlparse
+
+from twisted.internet import reactor
+from twisted.internet.defer import inlineCallbacks, returnValue, CancelledError
+from twisted.web.client import Agent, readBody
+
+
+TIMEOUT = 5  # seconds
```

Comment:
I usually indicate units by suffixing it with the variable name. Ex: `TIMEOUT_SECONDS`

Reply:
:+1: done